dts/bindings/sensor/st,lis2dw12-common.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2021 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+properties:
+    irq-gpios:
+      type: phandle-array
+      required: false
+      description: DRDY pin
+
```

Comment:
Since you are enhancing this driver, could you please fix up the multiline YAML string while you are here like this:

```
      description: |
        DRDY pin
```

Look at how the current format is rendered on one giant line here:

https://docs.zephyrproject.org/latest/reference/devicetree/bindings/sensor/st%2Clis2dw12-i2c.html#dtbinding-st-lis2dw12-i2c

Reply:
I see... OK, I'm making a global change for all properties. How to check it before pushing the commit? Should I run make htmldocs under doc?

Reply:
I used multi-line | in the YAML file for all properties. Boring enough I had to git rebase interactively and use 'e' one by one to change each commit for every property inserted. Not a problem anyway.

Thanks for suggestion.

Reply:
> Should I run make htmldocs under doc?

Yep, that's how you locally render documentation.